README.md

```diff
@@ -1,2 +1,256 @@
-# Hello-world
-This Repo is for code review
+<a name="readme-top"></a>
+
+<!--
+HOW TO USE:
+This is an example of how you may give instructions on setting up your project locally.
+
+Modify this file to match your project and remove sections that don't apply.
+
+REQUIRED SECTIONS:
+- Table of Contents
+- About the Project
+  - Built With
+  - Live Demo
+- Getting Started
+- Authors
+- Future Features
+- Contributing
+- Show your support
+- Acknowledgements
+- License
+
+OPTIONAL SECTIONS:
+- FAQ
+
+After you're finished please remove all the comments and instructions!
+-->
+
+<div align="center">
+  <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
+  <h3><b>Microverse README Template</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+-   [📖 About the Project](#about-project)
+    -   [🛠 Built With](#built-with)
+        -   [Tech Stack](#tech-stack)
+        -   [Key Features](#key-features)
+    -   [🚀 Live Demo](#live-demo)
+-   [💻 Getting Started](#getting-started)
+    -   [Setup](#setup)
+    -   [Prerequisites](#prerequisites)
+    -   [Install](#install)
+    -   [Usage](#usage)
+    -   [Run tests](#run-tests)
+    -   [Deployment](#deployment)
+-   [👥 Authors](#authors)
+-   [🔭 Future Features](#future-features)
+-   [🤝 Contributing](#contributing)
+-   [⭐️ Show your support](#support)
+-   [🙏 Acknowledgements](#acknowledgements)
+-   [❓ FAQ (OPTIONAL)](#faq)
+-   [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Hellow_World] <a name="about-project"></a>
+
+> Describe your project in 1 or 2 sentences.
+
+**[Hellow_World]** is a...
```

Comment:
- [ ] Kindly make sure you give a short description of your project as specified above in 1 or 2 sentences. In that way, a User reading your README will have an idea of the project.